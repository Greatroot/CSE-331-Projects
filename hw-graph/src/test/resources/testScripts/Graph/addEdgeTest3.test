# Test adding two edges, both overlapping | should return
true and add both edges

CreateGraph graph
AddNode graph n1
AddNode graph n2
AddEdge graph n1 n2 e1
AddEdge graph n1 n2 e2