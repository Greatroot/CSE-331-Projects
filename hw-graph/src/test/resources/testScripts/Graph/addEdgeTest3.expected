# Test adding two edges, both overlapping | should return
true and add both edges

created graph graph
added node n1 to graph
added node n2 to graph
added edge e1 from n1 to n2 in graph
added edge e2 from n1 to n2 in graph